### ISSUE TITLE
EncryptionResult.publickey is undefined

### ISSUE THREAD

**Issue by hems** [tracker1/cryptico-js#6]
On the readme you show an example with

````
cryptico.publicKeyID(EncryptionResult.publickey);
````

but the property .publickey doesn't exist, i believe you meant to write ````.cipher```` ?

**Comment by tracker1** [comment]
I'm frankly not sure...  I didn't change too much myself, but will look into it a bit... the readme is definitely inherited, so will need to try a couple things to be certain.

**Comment by hems** [comment]
i did test and .cipher is the right property you want to use!

**Comment by hems** [comment]
i didn't realise it was forked! what is the difference between your fork and the original?

**Comment by hems** [comment]
oh i just seen the original is PHP!

**Comment by hems** [comment]
yes have a try yourself but you will see that property doesn't exist! it's undefined!

have you used/is using this library currently?

**Comment by tracker1** [comment]
If you look at the network graph[0], my changes started around July 2014, I revised the build process (didn't like requiring unix tooling to build a node/JS project), other versions were meant for client-side more than node.js ... I had used this while I was experimenting with being able to share data between node and .net, and needed a way to create a key pair in a deterministic way from a shared secret at the time.  It's been a few years since I've used this though.

If you follow the network graph, there really haven't been many changes to this project along the way beyond some of the bugfixes that I brought in from outside.  I've updated the documentation readme per your suggestion above.  There isn't really a definitive version, and frankly I haven't delved deep into this, as I'm not really familiar enough with the underlying algorithms to comment on them.

[0] https://github.com/tracker1/cryptico-js/network
